plumgrid_playbooks/plumgrid_common.yml

```diff
@@ -20,9 +20,13 @@
     - plumgrid_common
     - create_ifcs
     - plumgrid_director
+  any_errors_fatal: yes
   vars:
+    pg_package_name: plumgrid-lxc
     pg_service_name: plumgrid
-  any_errors_fatal: yes
+    plumgrid_data_dir: "/var/lib/libvirt/filesystems/plumgrid-data"
```

Comment:
these shouldn't be defined, they will get picked automatically anywhere within the role using:

 pg_package[pg_package_type].plumgrid_data_dir 
pg_package[pg_package_type]. plumgrid_ifc_gateway_dir